tests/test_neuron_models.py

```diff
@@ -122,3 +122,66 @@ def test_morphology(self):
         n_locs = sum(len(c.points) for c in self.p75_pas.cables)
         self.assertEqual(len(self.p75_pas.cables), len(cell.sections), "missing cables")
         self.assertEqual(n_locs, sum(s.n3d() for s in cell.sections), "missing locs")
+
+
+@unittest.skipIf(
+    "NRN_SEGFAULT" in os.environ,
+    "These tests are skipped to test the other tests below separately. See https://github.com/neuronsimulator/nrn/issues/2641",
+)

```

Comment:
Has this been confirmed as required for this test suite?

Reply:
Actually, this is not necessary. I should remove it.